roles/ssh_host_signer/tasks/main.yml

```diff
@@ -52,6 +52,7 @@
     decrypt: true
   delegate_to: localhost
   connection: local
+  changed_when: false
```

Comment:
This is just a helper task to create a temporary copy of a file on your local ansible control host machine. It does  not change anything on the ansible targets. Removing the `changed_when: false` means the play will always report a change even when nothing changed on any of the ansible targets...

Reply:
I am lost? As far I understood: `changed_when: false` simply says that this task is **never** changed.
Without this line, I always got so far additional "changed" in in the summary of the playbook deployment - which is wrong in my opinion. The changed should appear **only** when something (remotely meaningful) is changed, and not every time playbook is (re)deployed.

Reply:
> changed_when: false means the play will always report a change

but I added this line, not removed one

Reply:
Sorry for the confusion. I got something messed up in git and then it looked you were removing that line instead of adding it. Adding `changed_when: false` makes sense.